package.json

```diff
@@ -42,6 +42,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "binary-search": "^1.3.2"
+    "binary-search": "^1.3.2",
+    "randombytes": "^2.0.3"
```

Comment:
Done, I replace the HEAD commit on my branch with a new one 1a9d51f that has no README change, only the index.html.